service/src/intTest/resources/application.yml

```diff
@@ -35,5 +35,5 @@ gp2gp:
     password: ${GP2GP_MONGO_PASSWORD:}
     options: ${GP2GP_MONGO_OPTIONS:}
     autoIndexCreation: ${GP2GP_MONGO_AUTO_INDEX_CREATION:true}
-    ttl: ${GP2GP_MONGO_TTL:P30D}
+    ttl: ${GP2GP_MONGO_TTL:PT3S}
```

Comment:
We need to update src/main/resources/application.yml to have a default of 7 days